server/core/src/config.rs

```diff
@@ -64,16 +64,22 @@ pub struct ServerConfig {
 }
 
 impl ServerConfig {
-    pub fn new<P: AsRef<Path>>(config_path: P) -> Result<Self, ()> {
+    pub fn new<P: AsRef<Path>>(config_path: P) -> Result<Self, &'static str> {
         let mut f = File::open(config_path).map_err(|e| {
             eprintln!("Unable to open config file [{:?}] 🥺", e);
+            "you aren't going to read this anyway"
```

Comment:
Why not just place holders then?

Reply:
because I was mad on power and trying to make clippy happy - have replaced these with `std::io::Error`